api_app/ingestors_manager/migrations/0025_ingestor_config_virustotal_example_query.py

```diff
@@ -0,0 +1,271 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+    ReverseManyToOneDescriptor,
+    ReverseOneToOneDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "virus_total.VirusTotal",
+        "base_path": "api_app.ingestors_manager.ingestors",
+    },
+    "schedule": {
+        "minute": "0",
+        "hour": "*",
+        "day_of_week": "*",
+        "day_of_month": "*",
+        "month_of_year": "*",
+    },
+    "periodic_task": {
+        "crontab": {
+            "minute": "0",
+            "hour": "*",
+            "day_of_week": "*",
+            "day_of_month": "*",
+            "month_of_year": "*",
+        },
+        "name": "VirusTotal_Example_QueryIngestor",
+        "task": "intel_owl.tasks.execute_ingestor",
+        "kwargs": '{"config_name": "VirusTotal_Example_Query"}',
+        "queue": "default",
+        "enabled": True,
+    },
+    "user": {
+        "username": "VirusTotal_Example_QueryIngestor",
+        "profile": {
+            "user": {
+                "username": "VirusTotal_Example_QueryIngestor",
+                "email": "",
+                "first_name": "",
+                "last_name": "",
+                "password": "",
+                "is_active": True,
+            },
+            "company_name": "",
+            "company_role": "",
+            "twitter_handle": "",
+            "discover_from": "other",
+            "task_priority": 7,
+            "is_robot": True,
+        },
+    },
+    "playbooks_choice": ["FREE_TO_USE_ANALYZERS"],
+    "name": "VirusTotal_Example_Query",
+    "description": "VirusTotal Ingestor example query taken from: https://blog.virustotal.com/2023/12/protecting-perimeter-with-vt.html",
+    "disabled": True,
+    "soft_time_limit": 60,
+    "routing_key": "ingestor",
+    "health_check_status": True,
+    "maximum_jobs": 30,
+    "delay": "00:00:30",
+    "model": "ingestors_manager.IngestorConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "virus_total.VirusTotal",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "hours",
+        "type": "int",
+        "description": "Download samples/IOCs that are up to X hours old",
```

Comment:
```suggestion
        "description": "Download samples that are up to X hours old",
```

better specify clearly the default operation